soft4pes/model/machine/induction_machine.py

```diff
@@ -5,15 +5,16 @@
 from types import SimpleNamespace
 import numpy as np
 from soft4pes.utils import dq_2_alpha_beta
+from soft4pes.model.common.system_model import SystemModel
 
 
-class InductionMachine:
+class InductionMachine(SystemModel):
     """
     Induction machine model operating at a constant electrical angular rotor speed.
     The state of the system is the stator current and rotor flux in the alpha-beta frame, i.e., 
-    [iS_alpha, iS_beta, psiR_alpha, psiR_beta]^T. The system input is the converter three-phase 
-    switch position. The initial state of the model is based on the stator flux magnitude reference 
-    and torque reference.
+    [iS_alpha, iS_beta, psiR_alpha, psiR_beta]^T. The machine is modelled with rotor flux alignment.
+    The system input is the converter three-phase switch position. The initial state of the model is 
```

Comment:
Should we say that the input is the three-phase switch position or that it is the converter output/stator voltage in abc?

Reply:
The state-space model used in this file includes the dc-link voltage so I would say that it is fine as it is

Reply:
Talking about converter output voltage is the same. Stator voltage is more general and covers the case of a converter

Reply:
Modulating signal is now mentioned in the docstring